themes/bootstrap3/js/common.js

```diff
@@ -649,6 +649,50 @@ function setupQRCodeLinks(_container) {
   });
 }
 
+// Useful function to delay callbacks, e.g. when using a keyup event
+// to detect when the user stops typing.
+// See also: https://stackoverflow.com/questions/1909441/how-to-delay-the-keyup-handler-until-the-user-stops-typing
+function getFacetListContentKeyupCallback() {
+  var timeout = null;
+  $('#ajax_contains').on('keyup', function onKeyupChangeFacetList() {
+    clearTimeout(timeout);
+    timeout = setTimeout(function onKeyupTimeout() {
+      getFacetListContent();
+    }, 500);
+  });
+}
+
+function getFacetListContent() {
+  let facet = $('#ajax_facet').val();
+  var url = VuFind.path + "/AJAX/JSON?q=sta&method=getFacetListContent&facet=" + facet;
+
+  let params = ['contains', 'sort', 'exclude', 'operator', 'page', 'limit', 'searchAction', 'urlBase'];
```

Comment:
Would it be more flexible to add a class or data-attribute to all the hidden inputs so we can query for them instead of listing them? (`[id^="ajax_"]` seems fragile)

Reply:
Like this?